internal/rbd/replicationcontrollerserver.go

```diff
@@ -437,6 +537,30 @@ func (rs *ReplicationServer) PromoteVolume(ctx context.Context,
 		}
 	}
 
+	var mode librbd.ImageMirrorMode
+	mode, err = getMirroringMode(ctx, req.GetParameters())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "failed to get mirroring mode %s", err.Error())
+	}
+	err = enableMirroringOnDummyImage(rbdVol, mode)
```

Comment:
For the second image that we promote, this enable will become a No-OP? IOW we need to disable/enable mirroring on the dummy image for each promote of a real image, such that the real image gets scheduled.

In the current form the second enable would become a no-OP is what I think. @BenamarMk ?

Reply:
Yes, correct its a no-op. the mirroring is enabled on dummy image only once for all promote operations. The same goes for disable also

Reply:
I think I tested that and it worked. I will test it again to see if refresh happens when the dummy image is enabled for mirroring while it is already ENABLED.  But if I am wrong, then we can always do this:

Promote PV:
```
1. disable mirroring Dummy
2. promote PV
3. enable mirroring Dummy
```

Demote PV:
```
1. disable mirroring Dummy
2. demote PV
```